pkg/controller/errors/errors.go

```
@@ -46,3 +46,16 @@ func (r RetryableError) Error() string {
 func IsUpdateConflict(err error) bool {
 	return strings.Contains(err.Error(), "the object has been modified; please apply your changes to the latest version")
 }
+
+// ShouldLog returns true if error should be logged.  This is used
+// when calling the Kubernetes API, so conflict and webhook
+// errors are not logged, the controller will just retry.
+func ShouldLog(err error) bool {
+	if err == nil {
+		return false
+	}
+	if IsUpdateConflict(err) || strings.Contains(err.Error(), "failed calling webhook") {
+		return false
```

what if there are legit issues with a webhook being called?

---

we will still retry.  Maybe I should log as error?

---

I think we need something...a throttled error message maybe.  Otherwise it will keep failing silently right?

---

I removed the webhook error, it will be displayed as an error if it happens